lib/getTrainingParameters.m

```diff
@@ -229,17 +229,34 @@
 cfg.grahn.patternSimple = getPatternInfo(grahnPatSimple, 'simple',cfg); 
 cfg.grahn.patternComplex = getPatternInfo(grahnPatComplex, 'complex', cfg); 
 
-% total sound duration proportion to gridIOI _/```\_  
-cfg.grahn.soundDurProp = 1; % 100% of gridIOI
-% onset ramp duration                       _/     
-cfg.grahn.eventRampon = 0.05; % 5% of gridIOI 
-% offset ramp duration                             \_ 
-cfg.grahn.eventRampoff  = 0.020; % 10% of gridIOI
-
 % the grid interval can vary across steps or segments (gridIOI selected 
 % randomly from a set of possible values for each new step or segment) 
 cfg.grahn.gridIOI = 0.190; 
 
+% Define envelope shape of the individual sound event. 
+% All parameters are defined in seconds. 
+
+% total sound duration _/```\_  
+cfg.grahn.soundDur             = 0.190; % s
+% onset ramp duration  _/     
+cfg.grahn.eventRampon          = 0.010; % s
+% offset ramp duration       \_ 
+cfg.grahn.eventRampoff         = 0.020; % s
+
```

Comment:
why don't we add this to getParams so both TapTraining and MainExp can use it? instead of having duplications here and there